catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -503,12 +503,12 @@ private Filter getHistoryFilter(Result result) {
                 .getMetacardType()
                 .getName();
         switch (typeName) {
-        case "deleted":
+        case "metacard.deleted":
```

Comment:
I assume there is no constant for this anywhere?  Is the type name just defined in blueprint?  What if the type name changes?